Formula/grpc.rb

```diff
@@ -0,0 +1,44 @@
+class Grpc < Formula
+  desc "Google's open-source RPC framework, based on protocol buffers."
+  homepage "http://www.grpc.io/"
+  head "https://github.com/grpc/grpc.git"
+
+  stable do
+    # The script run below, which is no longer needed in versions newer than
+    # 1.0.1, requires fun sed things, which don't seem to work with mac's sed.
+    url "https://github.com/grpc/grpc/archive/v1.0.1.tar.gz"
+    sha256 "efad782944da13d362aab9b81f001b7b8b1458794751de818e9848c47acd4b32"
+    depends_on "gnu-sed" => :build
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "protobuf"
+  depends_on "openssl"
+
+  def install
+    ENV["prefix"] = prefix
+    ENV["HAS_SYSTEM_PROTOBUF"] = "true"
+    ENV["HAS_PKG_CONFIG"] = "true"
+    ENV["HAS_EMBEDDED_PROTOBUF"] = "0"
```

Comment:
If any of these are needed only for a single `make` it'd be preferable to specify them only for that. e.g. `system "make", "install", "PREFIX=#{prefix}"`